### ISSUE TITLE
Build failure: bambu-studio

### ISSUE THREAD

**Issue by likarum** [NixOS/nixpkgs#293854]
### Steps To Reproduce

1. build *bambu-studio*

### Build log

[Gist](https://gist.github.com/likarum/2b142eb9d3b8bf4b3c25fdc5f215c7d9)

### Notify maintainers

@zhaofengli @norpol 

### Metadata

```console
❯ nix-shell -p nix-info --run "nix-info -m"
 - system: `"x86_64-linux"`
 - host os: `Linux 6.1.79, NixOS, 24.05 (Uakari), 24.05.20240303.b8697e5`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.18.1`
 - channels(root): `"nixos"`
 - nixpkgs: `/nix/store/bjvqq8c79dbi59g7xzcc6lhl0f19m3d7-source`

```

---

Add a :+1: [reaction] to [issues you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[issues you find important]: https://github.com/NixOS/nixpkgs/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc

**Comment by norpol** [comment]
Surprised to be tagged as a maintainer, I've already noticed the issue though and was already hovering opening a PR or issue myself. Use orca-slicer for the time being, build succeeds there. I already saw some patch used by Gentoo super-slicer that also addresses the compilation issue.

**Comment by GameDungeon** [comment]
> Use orca-slicer for the time being, build succeeds there. 

On my nix-os system orca-slicer fails to build too. I could send logs / open another issue. Don't know if I should given super-slicer and bambu already have issues open

**Comment by shadowpsi** [comment]
Appears to have been fixed:
 - system: `"x86_64-linux"`
 - host os: `Linux 6.8.1, NixOS, 24.05 (Uakari), 24.05.20240319.b06025f`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.18.2`
 - nixpkgs: `/nix/store/zaza7mgggz4m5h6z18kajabf4wciaj47-source`

**Comment by GameDungeon** [comment]
Orca is fixed for me now too.

**Comment by baracoder** [comment]
Works for me too. Should we close this issue?

**Comment by oddlama** [comment]
@likarum Can you confirm whether the segfault is solved by #331624 ?

**Comment by liberodark** [comment]
Now bambu-studio is build correctly you can close this old issue.
